features/bevragen/gezag/gezamenlijk-ouderlijk-gezag-bij-ouders-zonder-bsn.feature

```diff
@@ -0,0 +1,269 @@
+        | 000000024           | Jeanine Albers     |
+
+      Voorbeelden:
+      | zoek methode                                          |
+      | 'Gerda' wordt gevraagd                                |
+      | de personen ingeschreven op adres 'A1' wordt gevraagd |
+
+  @info-api @deprecated
+  Regel: wanneer gezag (fields veld bevat een string met waarde 'gezag') wordt gevraagd voor een minderjarige met één of twee ouders zonder burgerservicenummer, dan is het gezag niet te bepalen .
+
+    @info-api @deprecated
+    Scenario: minderjarige geboren na 1-1-2023 heeft twee ouders zonder burgerservicenummer
+      * heeft de volgende personen zonder burgerservicenummer als ouder
+        | voornamen (02.10) | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
+        | Jan               | Jansen                | M                           |
+        | Jeanine           | Albers                | V                           |
+      Als het 'gezag' veld van <zoek methode>
+      Dan heeft de response een persoon
+      En heeft de persoon een gezag
+        | type               | toelichting                                                   |
+        | GezagNietTeBepalen | Van één of beide ouders is de burgerservicenummer niet bekend |
+      * met minderjarige
+        | burgerservicenummer | naam.volledigeNaam | leeftijd |
+        | 000000012           | Gerda Jansen       | 2        |
+
+      Voorbeelden:
+      | zoek methode                                          |
+      | 'Gerda' wordt gevraagd                                |
+      | de personen ingeschreven op adres 'A1' wordt gevraagd |
+      
+    @info-api @deprecated
+    Scenario: minderjarige geboren na 1-1-2023 heeft twee ouders waarvan één zonder burgerservicenummer
+      * heeft de volgende persoon zonder burgerservicenummer als ouder
+        | voornamen (02.10) | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
+        | Jan               | Jansen                | M                           |
+      * heeft de volgende persoon als ouder
+        | burgerservicenummer (01.20) | voornamen (02.10) | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
+        | 000000024                   | Jeanine           | Albers                | V                           |
+      Als het 'gezag' veld van <zoek methode>
+      Dan heeft de response een persoon
+      En heeft de persoon een gezag
+        | type               | toelichting                                                   |
+        | GezagNietTeBepalen | Van één of beide ouders is de burgerservicenummer niet bekend |
+      * met minderjarige
+        | burgerservicenummer | naam.volledigeNaam | leeftijd |
+        | 000000012           | Gerda Jansen       | 2        |
+
+      Voorbeelden:
+      | zoek methode                                          |
+      | 'Gerda' wordt gevraagd                                |
+      | de personen ingeschreven op adres 'A1' wordt gevraagd |
+
```

Comment:
Denk je dat we nog moeten toevoegen dat naam van de ouders wel required is? Of staat dat in de features die we hebben gemaakt toen we (volledige) naam toevoegden? In de OAS staat het iig goed.

Reply:
Dit is een scenario om te begrijpen wat de functionaliteit (leveren van een ouder zonder burgerservicenummer) is. Ik weet niet of het kan voorkomen dat een ouder én geen naam (naam met alleen standaardwaarden) én geen burgerservicenummer heeft. Dit zou volgens mij als scenario moeten worden toegevoegd om te kunnen bepalen of de naam wel required kan zijn.